### DISCUSSION TITLE
New feature Request, Current/Total Layers on Display Screen

### DISCUSSION THREAD

**Discussion by chuckkay** [OpenNeptune3D/OpenNept4une#90] [Ideas]
Can we get curent layer/total layers on the display home screen when printing or if not room possibly a sub screen with even more information like how much filament is used/remaining to be used.....

**Comment by phillipthelen** [comment]
if you put the following in the display_connector.cfg file:

```
[print_screen]
z_display=layer
```

the printing screen will show the current layers instead of the height in mm :)

I still need to document the available config options somewhere.

**Reply by chuckkay** [reply]
thanks, but i tried changing height to Layer and it just stays at 0,
is Layer a preset variable already accounted for in your code or do i have to set something in slicer gcode?
start gcode
SET_PRINT_STATS_INFO TOTAL_LAYER=[total_layer_count]

after layer gcode
SET_PRINT_STATS_INFO CURRENT_LAYER={layer_num + 1}

when we figure this out can it be entered done so we can get something like this
[print_screen]
z_display=layer/total_layers
